tasks/main.yml

```
@@ -1,6 +1,10 @@
 ---
 - name: Gather Ansible Facts
-  ansible.builtin.setup: # If the full prefix isn't specified in Ansible 2.10+, we might end up running `ansible.windows.setup` instead.
+  ansible.builtin.setup:
+    gather_subset: "{{ datadog_ansible_facts_subset }}" # If the full prefix isn't specified in Ansible 2.10+, we might end up running `ansible.windows.setup` instead.
```

```suggestion
    gather_subset: "{{ datadog_ansible_facts_subset }}" 
    # If the full prefix isn't specified in Ansible 2.10+, we might end up running `ansible.windows.setup` instead.
```
line is too long, causes `ansible-lint` error